frontend/src/app/components/wp-single-view-tabs/tab/tab-wrapper.html

```diff
@@ -0,0 +1,6 @@
+<div class="detail-panel-description detail-panel--integration detail-panel--autocomplete-target"
+     *ngIf="workPackage">
+  <div class="detail-panel-description-content">
+    <ng-container #tabContent></ng-container>
```

Comment:
note: @oliverguenther and me wanted to enhance this part in a pairing session ℹ️  (could be in a separate PR though)

Reply:
Use something like this in the template instead of the whole `ngAfterViewInit`.

```
<ndc-dynamic
  *ngIf="tab"
  [ndcDynamicComponent]="tab.component"
  [ndcDynamicInputs]="{ workPackage: workPackage, tab: tab }"
  >
</ndc-dynamic>
```

Reply:
cool, thanks. I see we already use ndc-dynamic in other places 🎉  will do 🐜